### DISCUSSION TITLE
gear - lenses

### DISCUSSION THREAD

**Discussion by RadutoKomarotti** [tommi1hirvonen/ExifNotes#6] [Ideas]
Hello!

Some lenses have 1EV steps, but start at a different f-number. E.g. 1.7, 1.8, 2.5, 3.5 ... Currently it cannot be entered. Only when the step changes. But this is unnecessarily confusing and slow when editing. Would it be possible to modify the lens edit so that the step (e.g. 1EV) was preserved, but the first f-number was outside this scale?

Thaks

**Comment by cezary-zemis** [comment]
There are lenses with their maximum aperture not sticking to 1/3, 1/2 steps, e.g. Schneider Tele-Arton f/5.5, "converted" Schneider Symmar lenses f/12, Graflex Opter f/4.7 and so on. I suggest allowing setting maximum and minimum aperture to arbitrary value, and use steps within the range. Or, for some lenses, allowing arbitrary values for each shot (e.g. for Rodenstock Imagon with its steps H=5.7,  H=7.7, H=9.5, H=11.5).

**Comment by cezary-zemis** [comment]
There should also be a way to put arbitrary value of shutter time, e.g. when using B or T setting.
